### ISSUE TITLE
🐛 BUG: `npx wrangler@beta pages dev -- npm run dev` does not work?

### ISSUE THREAD

**Issue by leader22** [cloudflare/wrangler2#482]
### What version of `Wrangler` are you using?

0.0.17

### What operating system are you using?

Mac

### How to reproduce

```
# Create sandbox
mkdir wrangler-pages-repro
cd wrangler-pages-repro

# Setup frameworks
npm init svelte@next
# Hit enter key for all prompts
npm i
# Check npm run dev works properly

# Use wrangler pages dev command
npx wrangler@beta pages dev -- npm run dev
```

### Results

It just logs and quits.

```
Must specify a directory of static assets to serve or a command to run.
```

I'm not sure this is a bug or my own fault, because this is the most common way to use `wrangler@beta`. 😓 

But for my case,

https://github.com/cloudflare/wrangler2/blob/62fee9b9d35b00a75215252b142ec7e2ac4e2aaa/packages/wrangler/src/pages.tsx#L784

this `remaining` seems empty and fails to exec.

**Comment by eltorio** [comment]
Same error here
```bash
% npx wrangler@beta pages dev -- npm run serve
Must specify a directory of static assets to serve or a command to run.
```
Works perfectly with 0.0.16 tagged version

**Comment by petebacondarwin** [comment]
I have a fix for this: #502
